src/osm_osw_reformatter/osm2osw/osm2osw.py

```diff
@@ -16,36 +17,30 @@ def __init__(self, prefix: str, osm_file=None, workdir=None):
 
     async def convert(self) -> Response:
         try:
-            print('Estimating number of ways, nodes, points, lines, zones and polygons in datasets...')
```

Comment:
Why is this deleted?

Reply:
That is not need,  we are calculating it but not using anywhere